Dockerfile

```diff
@@ -1,11 +1,12 @@
 # syntax=docker/dockerfile:1
 
-FROM ghcr.io/imagegenius/baseimage-ubuntu:lunar
+FROM ghcr.io/imagegenius/baseimage-ubuntu:jammy
 
 # set version label
 ARG BUILD_DATE
 ARG VERSION
 ARG TORCH_VERSION
+ARG TORCHVISION_VERSION
```

Comment:
this is not going to be defined during build...

Reply:
well `TORCH_VERSION` is going to be passed through from build so might as well pin it to `${TORCH_VERSION}`, torch vision can be hard pinned in the dockerfile